### PR TITLE
segment import script

### PR DESCRIPTION
**Pull request by juniormp**

### CODE REVIEW

File: src/scripts/segment.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  const db = await Database.getInstance();
+  const transfersCollection = db.collection("transfers");
+
+  try {
+    const transfers = await transfersCollection.find().toArray();
+
+    const batch = transfers.map((transfer) => ({
+      type: "track",
+      userId: transfer.senderTgId,
+      event: "Transfer",
```
Comment (juniormp) [inline src/scripts/segment.js:72]: @tcoratger I believe this is the name of the transfer event.

Comment (tcoratger) [reply src/scripts/segment.js:72]: @juniormp Yes exact

File: src/scripts/segment.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      properties: {
+        Amount: transfer.tokenAmount,
```
Comment (tcoratger) [inline src/scripts/segment.js:74]: This should be `tokenAmount` to match what we currently have in FlowXO

File: src/scripts/segment.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        TxId: transfer.TxId,
+        chainId: transfer.chainId,
+        recipientTgId: transfer.recipientTgId,
+        recipientWallet: transfer.recipientWallet,
+        senderName: transfer.senderName,
+        senderTgId: transfer.senderTgId,
+        senderWallet: transfer.senderWallet,
+        tokenAddress: transfer.tokenAddress,
+        tokenAmount: transfer.tokenAmount,
```
Comment (tcoratger) [inline src/scripts/segment.js:83]: There is already the amount in the properties (first field)

File: src/scripts/segment.js (hunk 2 above)
Comment (juniormp) [reply src/scripts/segment.js:74]: @tcoratger You are right.
In segment there are 2 amounts (Amount and tokenAmount). The first one its incorrect. 

image

File: src/scripts/segment.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
+import {Database} from "../db/conn.js";
+import axios from "axios";
+
+const SEGMENT_API_ENDPOINT = "https://api.segment.io/v1/batch";
+const SEGMENT_WRITE_KEY = process.env.SEGMENT_WRITE_KEY;
```
Comment (tcoratger) [inline src/scripts/segment.js:5]: Can you just move that to our `constants.js` file?

Comment (juniormp) [reply src/scripts/segment.js]: sure 👍

**Pull request merged**